tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -954,14 +971,19 @@ void TRTEngineOp::ComputeAsync(OpKernelContext* ctx,
   if (!may_execute_native_segment()) {
```

Comment:
Maybe add `native_segment_absent_` to the definition of `may_execute_native_segment`, so that it is utilized here as well.